src/pages/locationsettings/index.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect } from "react";
+import Layout from "~/components/Layout";
+
+const index = () => {
+  interface IStadt {
+    name: string;
+    population: number;
+  }
+
+  const städte = [
+    { name: "Berlin", population: 1000000 },
+    { name: "Dresden", population: 510000 },
+    { name: "Dortmund", population: 200000 },
+    { name: "Dessau", population: 530000 },
+    { name: "Döbeln", population: 50000 },
+    { name: "Delb", population: 20000 },
+  ];
+  const [searchValue, setSearchValue] = useState("");
+  const [searchValue2, setSearchValue2] = useState("");
+  const [activeInput, setActiveInput] = useState(null);
```

Comment:
Add a type as a generic
```suggestion
  const [activeInput, setActiveInput] = useState<string>(null);